src/main/java/net/revelc/code/impsort/LineEnding.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.revelc.code.impsort;
+
+/**
+ * @author marvin.froeder
+ */
+public enum LineEnding {
```

Comment:
Since this code appears to be coming from `formatter-maven-plugin`, I think a comment should be added to indicate that. That way, in future, we will be able to more easily identify code which could be consolidated (by either depending on `formatter-maven-plugin` directly or by depending on a common library).

Reply:
Yes, I copied this file as well as the corresponding the test for it. The rest of the code is authored by me.

What sort of note do yo want? Do you want to just mention "formatter-maven-plugin" or provide a full link to the file (such as the GitHub permalink).

Reply:
Any link or reference that is sufficient to track it back to its source in case we want to grab an update, or consolidate code bases will be good. I'll let you decide what you think is best.